### PR TITLE
Update required tools for Ubuntu 14 install

### PR DESCRIPTION
**Pull request by chasetb**
Installing these items before creating the virtualenv solves issues during migration.

### CODE REVIEW

File: server/models.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ class Meta:
         unique_together = ("machine", "update")
 
 class Plugin(models.Model):
-    name = models.CharField(max_length=512, unique=True)
+    name = models.CharField(max_length=255, unique=True)
```
Comment (grahamgilbert) [inline server/models.py]: Why was this change needed?

Comment (chasetb) [inline server/models.py]: Django 1.8 performs a check (https://docs.djangoproject.com/en/1.8/ref/checks/#database) to make sure that “max_length” is no greater than 255 when using “unique=True” with MySQL. I do use MySQL so I changed it so I could push it my server with a “git pull.”

Otherwise, you get this error:
	SystemCheckError: System check identified some issues:

	ERRORS:
	server.Plugin.name: (mysql.E001) MySQL does not allow unique CharFields to have a max_length > 255.

Thanks,
Chase Thompson-Baugh
801.866.9174

**Comment by grahamgilbert** [comment]
On it's own, I don't think this is enough. A migration will be needed with something to truncate any that are longer than the max (I don't think there will be as this has only been out for a few days, but you never know).

**Comment by chasetb** [comment]
You're probably right. I'm still new at this and tackling issues as they come up.

**Comment by grahamgilbert** [comment]
So we can get this merged quicker, can you take out the changes for MySQL from this PR and put it into its own so we can discuss the best way forward. I can probably fix this, but not tonight.

**Comment by chasetb** [comment]
I pushed a change to the branch that puts the value back to “512”. https://github.com/chasetb/sal/commit/f05f3bf0a866b37862fea71652e3030dee1abc89 <https://github.com/chasetb/sal/commit/f05f3bf0a866b37862fea71652e3030dee1abc89> I’ll start up a new branch for my MySQL testing.
